bitnami/schema-registry/Chart.yaml

```diff
@@ -28,4 +28,4 @@ maintainers:
 name: schema-registry
 sources:
   - https://github.com/bitnami/charts/tree/main/bitnami/schema-registry
-version: 11.0.1
+version: 11.0.2
```

Comment:
As you are adding a new feature, and per [semver conventions](https://semver.org/#semantic-versioning-specification-semver), we should bump the chart's version to a new minor.